server/region_handler_test.go

```diff
@@ -282,6 +300,46 @@ func (ts *TidbRegionHandlerTestSuite) TestGetMvcc(c *C) {
 	err = decoder.Decode(&data2)
 	c.Assert(err, IsNil)
 	c.Assert(data2, DeepEquals, data)
+
+	// tests for normal index key
```

Comment:
Could we define an independent function to test index key?

Reply:
Done.